Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java

```diff
@@ -0,0 +1,16 @@
+public class StrongestRobot {
+    public static void main(String[] args) {
+        int[] weights = {300, 450, 200, 500, 350, 900, 0};
+
+        int strongestRobot = -1;
```

Comment:
not a mistake, but just wondering - why initializing to `-1`?

Reply:
Initially, I wanted to set the index to 0, but then I considered what would happen if the array were empty or all values were the same. In such cases, returning 0 would not be correct. So, I googled for a solution, and the recommendation was to use -1.
By the way, which option is more appropriate here?